### PR TITLE
[EDA] People & Groups: Contact Information Settings - Preferred Contact Information

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.js [hunk 1]
```diff
@@ -37,6 +37,14 @@ export default class SettingsSaveCanvas extends LightningElement {
             this.hierarchySettingsChanges.settingsSingleValueBySettingsName[hierarchySettingsChange.settingsName] =
                 hierarchySettingsChange.settingsValue;
         }
+        if (hierarchySettingsChange.settingsType === "boolean") {
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.js:40]: If we're going to go this far, then let's use a switch statement instead.

File: force-app/main/settingsComponentLibrary/lwc/settingsRowInput/settingsRowInput.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    @api value;
+    @api placeholder;
+    @api options;
+    @api disabled;
+    @api type;
+
+    handleInputChange(event) {
+        let eventDetail = {};
+
+        if (this.type === "toggle") {
```
Comment (MatthewBlanski) [inline force-app/main/settingsComponentLibrary/lwc/settingsRowInput/settingsRowInput.js:16]: Please change this to a switch statement to match the other conditionals we have below.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Only a few changes. The code is functional save for the odd bug with canceling the conditionally-shown element in this phone page.

File: force-app/main/default/classes/ContactInfoSettingsController.cls [hunk 3]
```diff
@@ -50,6 +50,22 @@ public with sharing class ContactInfoSettingsController {
...
     }
 
+    /********************************************************************************
+     * @description Retrieve a view model representing the preferred contact information
+     * settings specified in Hierarchy Settings and a list of fields of type "Phone" on
+     * Contact.
+     * @return A PreferredContactInfoSettingsVModel.
+     ********************************************************************************/
+    @AuraEnabled(cacheable=true)
+    public static PreferredContactInfoSettingsVModel getPreferredContactInfoSettingsVModel() {
```
Comment (MatthewBlanski) [inline force-app/main/default/classes/ContactInfoSettingsController.cls:60]: Where is the unit test for this?

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
Passes QA - Merging - Will Close WI

**Pull request merged**